components/src/core/CollapsableHeaderLayout/CollapsableHeaderLayout.tsx

```diff
@@ -51,7 +51,7 @@ export type CollapsibleLayoutProps = ViewProps & {
     HeaderProps: BLUIHeaderProps;
 
     /** Scroll component passed as a prop */
-    ScrollComponent?: (
+    scrollComponent?: (
```

Comment:
This should be capitalized since it represents a Component.

Reply:
yes I noticed that lint was not accepting ScrollComponent wanted camelcase. By the way what version of TS should we be using now? I believe in here I updated to 4.9.5...

Reply:
We can eslint-disable that...for props that represent components, we use the convention of using TitleCase (following MUI).

For typescript we should use the latest version that we can. I think they are up to 5 something now.